miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/InstituteController.java

```diff
@@ -76,23 +74,20 @@ public InstituteDto getInstitute(@PathVariable("id") Long id, UriComponentsBuild
   
   @RequestMapping(value = "/institute", method = RequestMethod.POST, headers = { "Content-type=application/json" })
   @ResponseBody
-  public ResponseEntity<?> createInstitute(@RequestBody InstituteDto instituteDto, UriComponentsBuilder uriBuilder) throws IOException {
+  public InstituteDto createInstitute(@RequestBody InstituteDto instituteDto, UriComponentsBuilder uriBuilder) throws IOException {
     Institute institute = Dtos.to(instituteDto);
     Long id = instituteService.create(institute);
-    UriComponents uriComponents = uriBuilder.path("/institute/{id}").buildAndExpand(id);
-    HttpHeaders headers = new HttpHeaders();
-    headers.setLocation(uriComponents.toUri());
-    return new ResponseEntity<>(headers, HttpStatus.CREATED);
```

Comment:
Should be replaced with `@ResponseStatus`. Same with `PUT` method in this class and similar methods in other REST controllers